drafts/draft-snell-multicodec-00.txt

```diff
@@ -0,0 +1,336 @@
+   | eth-tx-receipt     | ipld        | 0x95     | Ethereum            |
+   |                    |             |          | Transaction Receipt |
+   |                    |             |          | (RLP)               |
+   | eth-state-trie     | ipld        | 0x96     | Ethereum State Trie |
+   |                    |             |          | (Eth-Secure-Trie)   |
+   | eth-account-       | ipld        | 0x97     | Ethereum Account    |
+   | snapshot           |             |          | Snapshot (RLP)      |
+   | eth-storage-trie   | ipld        | 0x98     | Ethereum Contract   |
+   |                    |             |          | Storage Trie (Eth-  |
+   |                    |             |          | Secure-Trie)        |
+   | bitcoin-block      | ipld        | 0xb0     | Bitcoin Block       |
+   | bitcoin-tx         | ipld        | 0xb1     | Bitcoin Tx          |
+   | zcash-block        | ipld        | 0xc0     | Zcash Block         |
+   | zcash-tx           | ipld        | 0xc1     | Zcash Tx            |
+   | stellar-block      | ipld        | 0xd0     | Stellar Block       |
+   | stellar-tx         | ipld        | 0xd1     | Stellar Tx          |
+   | decred-block       | ipld        | 0xe0     | Decred Block        |
+   | decred-tx          | ipld        | 0xe1     | Decred Tx           |
+   | ipld-ns            | namespace   | 0xe2     | IPLD path           |
+   | ipfs-ns            | namespace   | 0xe3     | IPFS path           |
+   | swarm-ns           | namespace   | 0xe4     | Swarm path          |
+   | ipns-ns            | namespace   | 0xe5     | IPNS path           |
+   | ed25519-pub        | key         | 0xed     | Ed25519 public key  |
+   | dash-block         | ipld        | 0xf0     | Dash Block          |
+   | dash-tx            | ipld        | 0xf1     | Dash Tx             |
+
+
+
+Snell, et al.            Expires August 27, 2020                [Page 3]
+
+Internet-Draft                Multiformats                 February 2020
+
+
+   | swarm-manifest     | ipld        | 0xfa     | Swarm Manifest      |
+   | swarm-feed         | ipld        | 0xfb     | Swarm Feed          |
+   | udp                | multiaddr   | 0x0111   |                     |
+   | p2p-webrtc-star    | multiaddr   | 0x0113   |                     |
+   | p2p-webrtc-direct  | multiaddr   | 0x0114   |                     |
+   | p2p-stardust       | multiaddr   | 0x0115   |                     |
+   | p2p-circuit        | multiaddr   | 0x0122   |                     |
+   | dag-json           | ipld        | 0x0129   | MerkleDAG json      |
+   | udt                | multiaddr   | 0x012d   |                     |
+   | utp                | multiaddr   | 0x012e   |                     |
+   | unix               | multiaddr   | 0x0190   |                     |
+   | p2p                | multiaddr   | 0x01a5   | libp2p              |
+   | ipfs               | multiaddr   | 0x01a5   | libp2p (deprecated) |
```

Comment:
I'd be in favour of removing `ipfs` and only keep `p2p`. It will make things way easier for implementers if there's a 1:1 mapping between a name and a number.